jobs/kubelet/templates/bin/drain.erb

```diff
@@ -35,13 +45,45 @@ cordon_node() {
 }
 
 drain_node() {
-  kubectl --kubeconfig /var/vcap/jobs/kubelet/config/kubeconfig-drain drain \
```

Comment:
Can we set an environment variable for this script to point to this kubeconfig instead of pointing to it via the --kubeconfig path? The same path is duplicated a lot in this file?

Reply:
We will make this changes later and go ahead and merge it right now